src/common/config_parser.cpp

```diff
@@ -111,6 +112,9 @@ void ConfigParser::addOptionsGeneral(cli::CLIWrapper& cli) {
     "Print list of authors and exit");
   cli.add<bool>("--cite",
     "Print citation and exit");
+  cli.add<std::string>("--build-info",
+    "Print CMake cache variables and exit. Set to 'all' to print advanced variables")
```

Comment:
"CMake cache variables" is quite abstract. How about "CMake build options"?

Reply:
"CMake cache variables" was taken directly from the help message of "cmake -L", but I agree that "build options" may be more informative here.